plugins/genesys/fu-genesys-common.h

```diff
@@ -28,6 +48,51 @@ typedef struct __attribute__((packed)) {
 	guint8 firmware_version[4]; /* MMmm=MM.mm (ASCII string) */
 } FuGenesysStaticToolString;
 
+typedef struct __attribute__((packed)) {
+	guint8 running_mode; /* 'M' or 'C' */
```

Comment:
I forgot to ask. could you tell in one word what means 'M' and 'C'?

For example: `*/ 'M' for mask or 'C' for code*/`; any other values?

Reply:
what does mask mean exactly out of curiosity?

Reply:
> what does mask mean exactly out of curiosity?

An easy way to know is that the mask code was designed in a IC(fixed in hardware). Our IC can load a firmware from flash(bank code or ram code).